### DISCUSSION TITLE
syntax for chat documents

### DISCUSSION THREAD

**Discussion by odysseyalive** [n4ze3m/page-assist#96] [General]
I love this application. I was so happy to chuck text-generation-webui. 

In the Chrome Web Store description for Page Assist, it mentioned "Chat with Documents (pdf,csv,txt,md)". Is that the "Manage Knowledge" section in the options? If so, do these documents just become general knowledge, or is there syntax we can use to focus our queries on specific documents uploaded there?

**Comment by n4ze3m** [comment]
Hey, currently the knowledge base works like this: You can upload one or more documents and create a knowledge base. After processing, it will appear in the chatbox as a block icon where you can select the knowledge base and chat. 

Note that the quality of the RAG will differ compared to other services since Page Assist stores vectors in browser storage 


https://github.com/n4ze3m/page-assist/assets/39720973/e5d4a274-ba61-4f66-91ec-f5c4c1a8d07e
